build/dangerjs/detectNonApprovedDependencies/approvedPackages.ts

```diff
@@ -0,0 +1,69 @@
+export default [
+  '@babel/runtime@7.1.2',
+  '@babel/runtime@7.4.5',
+  'classnames@2.2.6',
+  'compute-scroll-into-view@1.0.11',
+  'css-in-js-utils@2.0.1',
+  'css-in-js-utils@3.0.0',
```

Comment:
Can we cleanup this file to have only actual dependencies there? In a follow up PR of course

Reply:
yes, we could - however, this file is intended to answer the following question: what are the packages that are approved? It is not this file's intent to reflect the dependencies that are 'actually' used - to save on maintainability costs. We may display actually used dependencies from the approved ones on each build, as part of DangerJS output instead